Gemfile

```diff
@@ -79,7 +79,7 @@ gem "secure_headers", "~> 2.5.0"
 
 # Frontend
 gem "backbone-on-rails", "~>0.9.10.0" # Legacy js
-gem "bootstrap", "~> 4.0.0.alpha3" # Bootstrap 4 - used for revised stylesheets
+gem "bootstrap"
```

Comment:
Should be locked to the version from [yarn.lock](https://github.com/bikeindex/bike_index/blob/master/yarn.lock)